skrub/_reporting/_summarize.py

```diff
@@ -45,6 +46,11 @@ def summarize_dataframe(
         Maximum number of rows from the end of the dataframe to show in the
         sample table.
 
+    verbose : int, default = 1
+        A switch that determines if messages show in the standard output
+        when report summary is being opened. Defaults to 1 to show messages.
+        Set it to 0 to avoid displaying messages in standard output.
```

Comment:
```suggestion
        Whether to print progress information while the report is being generated.

        * verbose = 1 prints how many columns have been processed so far.
        * verbose = 0 silences the output.
```

Just a nitpick suggestion to say that what we print is information about the progress, and it happens when creating the report even if we do not open it. Please feel free to reword it!

Reply:
Thank you  very much @jeromedockes. Was wondering why the tests were failing.

Also on the suggested changes to the documentation, I think we generalized it because setting  verbose = 1 also prints that extra message (flush = True) and not just the columns processed

Reply:
I see,  that makes sense! note the `print(flush=True)` is just to start a new line (move the terminal's cursor one line down so that the next printed thing doesn't get mangled with the report's output), it doesn't really print any content so we can think of it as wrapping up the display of progress.

Reply:
Right, that makes sense. Committing your suggestion.

Reply:
thanks! I'll also commit it in the TableReport's docstring in that case :)